frontend/src/translations/ru.global.json

```
@@ -0,0 +1,550 @@
+          "title": "Время безотказной работы"
+        },
+        "latest_produced_block": {
+          "text": "Высота блока, о которой узел проверки сообщил в самом последнем тактовом сообщении телеметрии.",
+          "title": "Последний блок"
+        },
+        "latest_telemetry_update": {
+          "text": "Телеметрия - это регулярное уведомление, поступающее от узлов, которое включает в себя общую информацию, такую как последняя известная высота блока и версия агента протокола NEAR (nearcore).",
+          "title": "Последнее обновление телеметрии"
+        },
+        "node_agent_name": {
+          "text": "Протокол NEAR может иметь несколько реализаций, поэтому агент — это имя этой реализации, где «near-rs» - это ${agent_name_url}.",
+          "url_text": "официальная реализация",
+          "title": "Имя агента узла"
+        },
+        "node_agent_version_or_build": {
+          "title": "Версия агента узла / Построение"
+        }
+      },
+      "ValidatorMetadataRow": {
+        "pool_info": {
+          "website": "Сайт",
+          "email": "Email",
+          "twitter": "Twitter",
+          "discord": "Discord",
+          "description": "Описание"
+        },
+        "pool_info_tip": {
+          "text": "Если вы владелец узла, не стесняйтесь заполнить всю ${pool_info_url}, чтобы продвигать собственный узел!",
+          "url_text": "связанную информацию"
+        }
+      },
+      "NodesEpoch": {
+        "current_epoch_start": "Начало текущей эпохи",
+        "block": "Блок",
+        "complete": "завершено",
+        "remaining": "осталось"
+      },
+      "NodesContentHeader": {
+        "nodes": "Узлы"
+      },
+      "NodeNav": {
+        "validating": "Подтверждение",
+        "proposed": "Предложено"
+      },
+      "NodesCard": {
+        "nodes_validating": "Проверка узлов",
+        "total_supply": "Общее предложение",
+        "total_stake": "Всего в стейкинге",
+        "seat_price": "Стоимость места валидатора"
+      },
+      "Nodes": {
+        "validator": "Валидатор"
+      },
+      "NodeRow": {
+        "online": {
+          "title": "В сети",
+          "text": "Узлов в сети"
+        }
+      }
+    },
+    "transactions": {
+      "Transactions": {
+        "no_transactions": "Нет транзакций"
+      },
+      "ActionMessage": {
+        "CreateAccount": {
+          "new_account_created": "Создан новый аккаунт: "
+        },
+        "DeleteAccount": {
+          "delete_account": "Удалить аккаунт ",
+          "and_transfer_fund_to": " и перевести оставшиеся средства на "
+        },
+        "DeployContract": {
+          "contract_deployed": "Развёрнутый контракт: "
+        },
+        "FunctionCall": {
+          "called_method": "Вызываемый метод: '${method_name}' в контракте: ",
+          "arguments": "Параметры:"
+        },
+        "Transfer": {
+          "transferred": "Переведено "
```

to avoid "to" on russian version
```suggestion
          "transferred": "Переведено ",
          "to": "  "
```